products/statement-generator/public/locales/en/translation.json

```diff
@@ -191,67 +191,59 @@
+    "paragraph1.1c": "Expungement lifts many of the negative effects of a criminal conviction and can give you a fresh start in your life.",
+    "title1.2": "What is a declaration letter?",
+    "paragraph1.2a": "A declaration letter in California is a written statement in a petition for expungement or reduction of some infractions, misdemeanors, or felonies. It may be required or may be added as supporting evidence depending on your case.",
+    "paragraph1.2b": "It is a written letter that explains why dismissal of your conviction would be in the interest of justice. It allows you to tell the judge your story and describe your circumstances, efforts to reform, and motivation for expungement or reduction. It is often about one page long.",
+    "title1.3": "Why does reduction or expungement matter to me?",
+    "paragraph1.3a": "Expungement means that in most cases your past criminal conviction cannot be used against you when applying for a job, housing, state licensing, or student loans. You may also regain some of your rights, like, in some cases, owning a firearm.",
+    "paragraph1.3b": "Reduction, also called “reclassification” or “redesignation”, allows you to change a past felony conviction to a misdemeanor, or a misdemeanor to an infraction, even if you have served a sentence. Certain legal provisions allow a felony conviction to be reduced to a misdemeanor and then be expunged.",
+    "title1.4": "What is the expungement process in California?",
+    "paragraph1.4a": "To learn more about the expungement process in California and eligibility, please check the following California Courts and California Legislature resources:",
+    "paragraph1.4b": "<a href='https://selfhelp.courts.ca.gov/clean-your-record' target='_blank' rel='noreferrer'>California Records Self-Help Guide: Clean your record</a>",
+    "paragraph1.4c": "<a href='https://www.courts.ca.gov/partners/documents/ExpSBS.pdf' target='_blank' rel='noreferrer'>Clearing Your Adult Criminal Record in California: A Step-by-Step Guide</a>",
+    "paragraph1.4d": "<a href='https://leginfo.legislature.ca.gov/faces/billTextClient.xhtml?bill_id=202120220SB731' target='_blank' rel='noreferrer'>California Senate Bill 731</a>",
+    "section2": "Using Expunge Assist",
+    "title2.1": "What is Expunge Assist?",
+    "paragraph2.1a": "<p className={classes.manualText}>Expunge Assist is a tool that helps people with eligible criminal convictions in California write a declaration letter for expungement or reduction petitions. It serves as a guide that takes you through the writing process step by step. It is a free service provided by <a href='https://www.hackforla.org/' target='_blank' rel='noreferrer' style={{ textDecoration: 'underline' }}>Hack for LA, </a> a project under the nonprofit <a href='https://www.civictechstructure.org/' target='_blank' rel='noreferrer' style={{ textDecoration: 'underline' }}>Civic Tech Structure</a>.</p>",
```

Comment:
adding the material ui class here was a clever way to try to solve this, but because material ui is a compiled styling language (aka the css version of react) and the translation.json file is in the public folder, this gets passed literally as `classes.manualText` Basically, this text gets added _after_ the app is built so it's not able to translate `classes.manualText` into a css class.

<img width="450" alt="Screen Shot 2023-10-24 at 2 18 20 PM" src="https://github.com/hackforla/expunge-assist/assets/75542938/605873ca-3cb9-443f-99f0-75e82cf7a710">

Lets remove this and double check that the intended styles are present.

Reply:
Note! this comment only applies to the custom material UI classes. I noticed you used inline styling below and that will render correctly because you're 'forcing' html onto the page, so there's nothing to translate.

Reply:
Ah, yes. Thank you for explaining! I accidentally copied that over from when I was using the manual text instead of the dangerouslySetHTML. It looks like it was just the one so I took care of it!

Reply:
@sydneywalcoff Hi! I'm trying to fix the jump down, and managed to make it a much smaller jump, but I'm struggling to override the default jump since it is built into the MUI accordion itself. I noticed the accordions on the bottom of the EA homepage also jump down in the same way when they're opened, so I thought maybe the ones on the FAQ should function the same. What are your thoughts?

Reply:
@whitneywind oh good call out. Let's leave it then, and maybe create a ticket to fix this later.

Reply:
Okay, got it! I'll just push up the other change then